domains/quantummapleqc.json

```
@@ -0,0 +1,12 @@
+{
+    "description": "so this is about showing my projects and gonna add more stuff soon!",
+    "repo": "https://github.com/QuantumMapleQC/quantumapleqc.github.io",
+    "owner": {
+        "username": "QuantumMapleQC",
+        "email": "thefattestmod@gmail.com",
+        "Discord": "quantummapleqc"
+    },
+    "record": {
+        "CNAME": "https://quantummapleqc.github.io/quantumapleqc.github.io/"
```

```suggestion
        "CNAME": "quantummapleqc.github.io"
```
CNAMEs cannot contain protocols (`https://`) or slashes (`/`).